packages/terra-slide-panel/src/terra-dev-site/doc/example/DefaultSlidePanel.jsx

```diff
@@ -2,14 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
 import SlidePanel from 'terra-slide-panel/lib/SlidePanel';
+import classNames from 'classnames/bind';
+import styles from './DefaultSlidePanel.scss';
+
+const cx = classNames.bind(styles);
 
 const mainContentForSlidePanel = togglePanelHandler => (
   <div>
-    <header style={{ backgroundColor: 'lightgrey' }}>
-      <h3 style={{ margin: '0px', padding: '5px', display: 'inline-block' }}>Main Content</h3>
-      <button id="mainToggleBtn" type="button" onClick={togglePanelHandler} style={{ display: 'inline-block' }}>Main Toggle Panel</button>
+    <header className={cx('header-content')}>
+      <h3 className={cx('heading')}>Main Content</h3>
```

Comment:
just curious to know why we are not using combinators like 
``` .header-content h3 { 
  ---style-code---
} ``` 
here instead of having separate classes for each element.

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.